packages/webapp/src/components/Finances/Transaction/ExpandedContent/styles.module.scss

```diff
@@ -38,3 +37,30 @@ button.toDetail {
     margin-left: 8px;
   }
 }
+
+.labourTable {
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+  background: #f6fbfa;
```

Comment:
I'd remove this background because it's already taking the background color from the parent container, and this one's causing an odd bit of green to show up when hovering over the item